Automattic/jetpack
# VideoPress: switch privacy icon according to video privacy

Fixes https://github.com/Automattic/jetpack/issues/26664

#### Changes proposed in this Pull Request:

* VideoPress:
  * Replace padlocks icons with eye ones
  * Stich the privacy icon when video privacy changes


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to VideoPress admin dashboard
* Confirm the padlocks icons are replaced with the eye ones
* Confirm the card switches the icon when the video privacy changes 

https://user-images.githubusercontent.com/77539/194324396-79a52efe-5a3f-4dbc-9b4c-b3498e985be7.mov
